code/datums/outfits/antagonist/mercenary.dm

```diff
@@ -8,6 +8,8 @@
 	shoes = /obj/item/clothing/shoes/magboots/merc
 	pda_slot = slot_belt
 	pda_type = /obj/item/modular_computer/pda
+	back = /obj/item/storage/backpack/satchel/military
+	//I gave them a satchel. I don't fucking know why they didn't have one before. -Vandu
```

Comment:
they spawn without one because they immediately go to the loadout room and take one from there (or the backpac

Reply:
Yeah, which is kind of dumb? Why are they the one people who don't spawn with anything for a backpack? Like seriously, it's painful
They spawn with their survival box in their HAND for crying out loud. I really think they should just spawn with a backpack, it seems like a useless inconvenience and an ancient bay leftover for them not to.

Reply:
> Why are they the one people who don't spawn with anything for a backpack?

Because they get their full loadout in a different room.

Reply:
I'm saying they shouldn't. They should get their satchel on spawn. I can change the map to remove the satchels from the room, if you really want.